SEED-platform/seed
# CSV Export Header

#### Any background context you want to provide?
CSV column header was not showing display names

#### What's this PR do?
* Show display names in CSV header
* Fix exporting on tax lot page

#### How should this be manually tested?
* Import some data
* Select some items, then go to export
* Verify columns that are selected are exported with the correct headers
* Do the same on tax lots

#### What are the relevant tickets?
#1543 #1512 

#### Screenshots (if appropriate)
![screen shot 2018-01-18 at 9 46 50 am](https://user-images.githubusercontent.com/1907354/35112408-3079cd26-fc3b-11e7-8f5b-a27f2736d2a5.png)
![screen shot 2018-01-18 at 9 36 07 am](https://user-images.githubusercontent.com/1907354/35112409-3094542a-fc3b-11e7-8f7c-b01783accea5.png)

#### Definition of Done:
